### DISCUSSION TITLE
Motivation for new participants

### DISCUSSION THREAD

**Discussion by ivanturlakov** [arriven/db1000n#506] [Ideas]
To attract a large number of new participants, you can use the approach often used by new blockchain projects. In short, you need to measure the activity of participants in the project in order to reward the best participants in the future, it can be project tokens, nft's etc.

**Comment by arriven** [comment]
There's already a project following that philosophy - disbalancer.com (they're also using their tech to help Ukraine right now)

Adding blockchain to my project would overcomplicate it and defeat the purpose of having simple opensource tool that simplifies stress testing some infra against DDoS
